app/models/patient.rb

```diff
@@ -48,6 +48,31 @@ class Patient < ApplicationRecord
 
   validates :assigned_user, numericality: { only_integer: true, allow_nil: true, greater_than: 0, less_than_or_equal_to: 9999 }
 
+  validates :last_date_of_exposure, earliest_date: { date: Date.new(2020, 1, 1) },
```

Comment:
So Rails just automatically includes the concern and matches it based on the name here, right?